### PR TITLE
Create Address book

### PR DESCRIPTION
**Pull request by yuriko627**
to add your fav user/address

### CODE REVIEW

File: src/pages/address_book.js [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+              <td className="default-address">
+                e.g. 0x0000000000000000000000000000000000000000
+              </td>
+              <td className="default-remove-button">
+                <FontAwesomeIcon icon="trash" />
+              </td>
+            </tr>
+          ) : (
+            props.addressLists.map(addressList => (
+              <AddressListItem
```
Comment (syuhei176) [inline src/pages/address_book.js]: Is `AddressListItem` a item in addressList?

Comment (yuriko627) [reply src/pages/address_book.js]: `AddressLists` is an array that has `AddressList` objects as its elements. `AddressListItem` is just a name for the component that renders one of the AddressList objects in one Address book row

Comment (syuhei176) [reply src/pages/address_book.js]: Thanks. So, `AddressList` isn't list of multiple addresses and is just corresponding to one account, right?

Comment (yuriko627) [reply src/pages/address_book.js]: Right! Now that I think there should be one `AddressList` array that has `AddressListItem` objects as its elements, where `AddressListItem` has 3 keys `id`,`name` and `address`. Making list plural like `AddressLists` is weird when there is only one list needed in the page.

**Pull request merged**